cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
```

Comment:
Ensure that the `Movie` model and `MovieSerializer` are correctly defined in their respective files, as they are crucial for the functionality of these views.